### DISCUSSION TITLE
Is there a way to disable postUpdateOptions for a repository?

### DISCUSSION THREAD

**Discussion by Keysox** [renovatebot/renovate#15740] [Request Help] [answered]
Right now, we have a `renovate.json` file that references a shared config. The shared `default.json` has `postUpdateOptions` set and because it's `mergeable`, setting it to `[]` in the repo doesn't seem to work. Any ideas or could I create an Issue for this? Thanks!

**Comment by viceice** [comment, marked as answer]
did you tried to set it to `null`?

**Reply by Keysox** [reply]
`"postUpdateOptions": null` did the trick. Thank you!
